openedx/stanford/djangoapps/register_cme/admin.py

```diff
@@ -5,4 +5,35 @@
 
 from .models import ExtraInfo
 
-admin.site.register(ExtraInfo)
+
+class ExtraInfoAdmin(admin.ModelAdmin):
+    """
+    Admin interface for ExtraInfo model.
+    """
+
+    readonly_fields = (
+        'user',
+    )
+
```

Comment:
Let's remove the blank lines between instance variables,
but leave them elsewhere.

```python
list_display = (
    # ...
)
readonly_fields = (
    # ...
)
search_fields = (
    # ...
)
```